extensions/blocks/anchor-fm/editor.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Internal dependencies
+ */
+import { name } from '.';
+import getJetpackExtensionAvailability from '../../shared/get-jetpack-extension-availability';
+import { dispatch } from '@wordpress/data';
+import { createBlock } from '@wordpress/blocks';
+
+/*
+ * Register the main "anchor-fm" extension.
+ */
+const isExtensionAvailable = getJetpackExtensionAvailability( name )?.available;
+
+if ( isExtensionAvailable ) {
+	/*
+	 * Insert link badge if needed.
+	 */
+	const episode = window.Jetpack_AnchorFm_Episode;
+	if ( typeof episode === 'object' ) {
+		window.onload = () => {
+			dispatch( 'core/block-editor' ).insertBlocks(
+				[
+					createBlock( 'core/image', {
+						url: 'https://cldup.com/Dv6JZWyRpq-1200x1200.png',
```

Comment:
Where can we host this officially? I think it should be loaded from some global CDN address and not from the specific Jetpack directory of the current site, since these links will break after the plugin gets deactivated. They should work on their own, since they are a simple image block. 

I haven't found a good place where Spotify hosts these and can be hotlinked. I've only been able to download this badge as a ZIP file from the dashboard.

Reply:
@sfougnier Do you have access to the "official" spotify badge? Is this something we'd want to have a SVG?

Reply:
Here is the official archive I got from the dashboard https://podcasters.spotify.com/resources/spotify-podcast-badges.zip. It includes SVGs too. We just probably need to pick one/more and host it somewhere. Maybe widgets.wp.com if there is nothing more suitable?